.devcontainer/install.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
+
+set -eux
+
+# Add a welcome message
+echo '👋 Welcome to Roots Development 🌱' | sudo tee /usr/local/etc/vscode-dev-containers/first-run-notice.txt
+
+WORKSPACE_FOLDER="${WORKSPACE_FOLDER:-"${PWD##*/}"}"
+
+# This is the default .env file used by the application
+FALLBACK_APP_ENV_FILE="${WORKSPACE_FOLDER}/.devcontainer/config/app/.env.example";
+APP_ENV_FILE="${FALLBACK_APP_ENV_FILE}";
+
+# If .env file exists in the workspace, let's use that instead
+if [ ! -z ${APP_ENV+x} ] && [ -f "${WORKSPACE_FOLDER}/.devcontainer/.env.${APP_ENV}" ]; then
+  APP_ENV_FILE="${WORKSPACE_FOLDER}/.devcontainer/.env.${APP_ENV}";
+elif [ -f "${WORKSPACE_FOLDER}/.devcontainer/.env" ]; then
+  APP_ENV_FILE="${WORKSPACE_FOLDER}/.devcontainer/.env";
+fi
+
+. "${APP_ENV_FILE}";
+# source our application env vars
+
+REPOSITORY_URL="${REPOSITORY_URL:-'https://github.com/roots/bedrock.git'}"
+
+sudo chown -R vscode:www-data /roots
+cd /roots/app
+
+# if composer.json already exists, exit early
+if [ -f "composer.json" ]; then
+  exit 0
+fi
+
+# Clone the repository
+git clone --depth=1 ${REPOSITORY_URL} . \
+  && rm -rf .git
```

Comment:
This could _maybe_ be an issue if `REPOSITORY_URL` points to a path on the local file system. I'm not sure; I haven't tested that.